README.md

```diff
@@ -1,8 +1,55 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Welcome to Mutual-Aid-Project! This site is for organizing mutual aid efforts within your community. 
 
 ## Setup
 
-Environmental variables:  
+Before starting, please reach out to Megan at info@townhallproject.com for environmental variables. 
+Once you receive this information please create an .env file at the root level (same level as .gitignore) 
+and copy and paste information into that file. 
+
+#### The general development workflow is as follows:
+- Fork the Mutual-Aid-project admin repo
+- Make changes to your forked repo
+- Send PRs from the forked repo to the main Mutual-Aid-project repo
+
+#### Merging your changes to the main Mutual-Aid-project repo
+1. update your forked master branch to match the main Mutual-Aid-project master branch:
+- Set up the original Mutual-Aid-project admin repo as a remote (this only has to be done once):
+  `git remote add upstream https://github.com/townhallproject/admin.git`
+  - To verify that the remote was added: `git remote -v`
+- `git checkout master`
+- `git pull upstream master`
+  - this brings all new content from the main Mutual-Aid-project repo into the current branch of your cloned repo
+- `git push origin master`
+  - this sends those changes to your fork
+
+2. Rebase your topic branch onto your fork's master, which is now up to date with everyone elses changes:
+- `git checkout <your branch>`
+- `git rebase master`
+  - if there are any problems they will become apparent here
+- `git push origin master`
+
+3. Create a pull request to the main Mutual-Aid-project repo master
+- use the github UI
+
+1. Download the files
+Clone this repository to a folder of your choice on your local machine: git clone <repo URL>
+Navigate to the newly cloned repository folder
+Install the project dependencies: npm i
+Run npm start to open in localhost
+  
+5. Devtools setup
+React and Redux devtools can be helpful tools to install for debugging in Chrome:
+
+React devtools chrome extension
```

Comment:
hyperlink to the tools. Also use bullets to list them out

Reply:
bullets and hyperlinks.